### PR TITLE
Support http and https metrics

### PR DESCRIPTION
**Pull request by spmcbride1201**
This pull request includes improved utility functions and charts designed to support display of both http and https metrics.

### CODE REVIEW

File: src/components/HTTPConnectionsAreaChart.js [hunk 1]
```diff
@@ -1,30 +1,36 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
-import { AreaChart, Area, CartesianGrid, XAxis, YAxis, Tooltip, ResponsiveContainer } from 'recharts';
+import { LineChart, Line, CartesianGrid, XAxis, YAxis, Tooltip, ResponsiveContainer } from 'recharts';
```
Comment (tilleryd) [inline src/components/HTTPConnectionsAreaChart.js]: Perhaps this file should be renamed to something generic or `HTTPConnectionsLineChart.js`.

File: src/components/HTTPStats.js [hunk 2]
```diff
@@ -1,20 +1,36 @@
...
+        </thead>  
+        <tr>
+          <td>HTTP</td>  
+          <td>{totalHTTPRequests}</td>
+          <td>{totalSuccessfulHTTPRequests} </td>
+        </tr>
+        <tr>
+          <td>HTTPS</td>  
+          <td>{totalHTTPSRequests}</td>
+          <td>{totalSuccessfulHTTPSRequests} </td>
```
Comment (tilleryd) [inline src/components/HTTPStats.js]: Do the additional spaces for the total successful requests for HTTP and HTTPS perform some visual function?  Or are they benign and thus can be removed?

File: src/components/HTTPConnectionsAreaChart.js (hunk 1 above)
Comment (spmcbride1201) [inline src/components/HTTPConnectionsAreaChart.js]: Just finished renaming all of the charts and imports to not include Line or Area. Pretty nuts that I was doing that, so thanks for calling me out.

File: src/components/HTTPStats.js (hunk 2 above)
Comment (spmcbride1201) [inline src/components/HTTPStats.js]: benign... and removed

**Pull request merged**